package.json

```diff
@@ -295,6 +295,7 @@
     "@metamask/design-tokens": "^4.2.0",
     "@metamask/ens-controller": "^15.0.0",
     "@metamask/ens-resolver-snap": "^0.1.2",
+    "@metamask/eth-block-tracker": "^11.0.3",
```

Comment:
Hmm. Given that we're not using this package, but a peer dependency suggests we should be using it, this suggests that it was listed as a peer dependency in error.

Perhaps we can ignore the warning for now, and remove it in `@metamask/transaction-controller` or at least track the problem with an issue.

Reply:
Sounds good, I just did remove it, I will remove it from peerDeps in `@metamask/transaction-controller` too